### PR TITLE
[Improve][Connector-V2] Change `amazonsqs` to `AmazonSqs` as connector identifier

### PR DESCRIPTION
**Pull request by Hisoka-X**
### Purpose of this pull request
Uniformly use `AmazonSqs` instead of `amazonsqs` as the connector identifier, and add AmazonSqs to the binary package.

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
exist test.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @HaoXuAI

**Review by wuchunfu — APPROVED** [review]

File: seatunnel-connectors-v2/connector-amazonsqs/src/main/java/org/apache/seatunnel/connectors/seatunnel/amazonsqs/sink/AmazonSqsSink.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public class AmazonSqsSink extends AbstractSimpleSink<SeaTunnelRow, Void> {
 
     @Override
     public String getPluginName() {
-        return "amazonsqs";
+        return "AmazonSqs";
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-amazonsqs/src/main/java/org/apache/seatunnel/connectors/seatunnel/amazonsqs/sink/AmazonSqsSink.java:53]: Is it compatible with older versions?

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-amazonsqs/src/main/java/org/apache/seatunnel/connectors/seatunnel/amazonsqs/sink/AmazonSqsSink.java:53]: AmazonSqs not released in 2.3.3. PS: our plugin name mapping ignore case.

**Comment by HaoXuAI** [comment]
thanks!

**Review by hailin0 — APPROVED** [review]

File: seatunnel-connectors-v2/connector-amazonsqs/src/main/java/org/apache/seatunnel/connectors/seatunnel/amazonsqs/sink/AmazonSqsSink.java (hunk 1 above)
Comment (hailin0) [reply seatunnel-connectors-v2/connector-amazonsqs/src/main/java/org/apache/seatunnel/connectors/seatunnel/amazonsqs/sink/AmazonSqsSink.java:53]: good

**Pull request merged**